include/simplecameralib/Camera_reader.h

```diff
@@ -0,0 +1,61 @@
+#include <opencv2/opencv.hpp>
+#include <iostream>
+
+using namespace cv;
```

Comment:
"using namespace..." is not recommended. Better to use scope use scope operator :: wherever required

Reply:
OK, Thanks